feature-libs/quote/components/quote-confirm-action-dialog/quote-confirm-action-dialog.component.html

```diff
@@ -0,0 +1,107 @@
+<div
+  #dialog
+  class="cx-modal-container"
+  [cxFocus]="focusConfig"
+  (esc)="dismissModal('Escape clicked')"
+>
+  <div
+    class="cx-modal-content"
+    *ngIf="confirmationContext$ | async as confirmationContext"
+  >
+    <!-- Modal Header -->
+
+    <div class="cx-dialog-header modal-header">
+      <div class="cx-dialog-title modal-title">
+        {{
+          confirmationContext.title
+            | cxTranslate: { code: confirmationContext.quote.code }
+        }}
+      </div>
+      <button
+        type="button"
+        class="close"
+        [attr.aria-label]="'common.close' | cxTranslate"
+        (click)="dismissModal('Cross click')"
+      >
+        <span aria-hidden="true">
+          <cx-icon [type]="iconTypes.CLOSE"></cx-icon>
+        </span>
+      </button>
+    </div>
+
+    <!-- Modal Body -->
+    <div class="cx-dialog-body modal-body">
+      <div class="cx-dialog-row">
+        <div class="cx-dialog-item">
+          <div class="quote-name-description">
+            <div class="quote-name">
+              <div class="heading">
```

Comment:
do we need this deep HTML-Tree? please try to save some divs here. 
style clases cx-dialog-row, cs-dialog-item, quote-name-description seem not to be used, maybe we can remove those divs